packages/node/src/entity/Node.ts

```diff
@@ -0,0 +1,12 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+
+@Entity()
+export class Node {
+  @PrimaryColumn()
+  // @ts-ignore
```

Comment:
Typescript doesn't seem to like these declarations.

Reply:
https://github.com/typeorm/typeorm/issues/2936 found a weird workaround, the issue is the `strictPropertyInitialization` flag in the tsconfig.